neo/tools/compilers/roqvq/roqParam.cpp

```diff
@@ -108,8 +108,8 @@ void roqParam::InitFromFile( const char *fileName )
 			readarg++;
 			continue;
 		}
-// scaleable
-		if (token.Icmp( "scaleable") == 0) {
+// scalable
+		if (token.Icmp( "scalable") == 0) {
```

Comment:
this looks like a very bad idea. apparently some kind of input file is expected to have this typo, changing this here will break parsing.